App/Components/AsyncImage.js

```diff
@@ -53,3 +67,16 @@ export default class AsyncImage extends React.Component {
     }
   }
 }
+
+const mapStateToProps = (state, ownProps) => {
```

Comment:
You don't have to make `AsyncImage` a redux connected component like this. You can derive this state>props for the node state up in `TextilePhotos`, then you just pass the `nodeState` as a prop into `PhotoGrid` and them again as a prop into `AsyncImage`.